Bottom: 302d33d5bb32c5145371cfd44c4a21d5852e6c1f
Top:    4642fd5d9551f510facefb874421a15b49150ffc
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-14 19:52:28 +0000

Refresh of add-flag-parameters-to-resetfo

---

diff --git a/src/find.c b/src/find.c
index 274a0f5..d692cc1 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3294,7 +3294,7 @@ LookupUnusedPins (FILE * FP)
 {
   /* reset all currently marked connections */
   User = true;
-#warning WHAT IS THEFLAG SET TO??
+  TheFlag = FOUNDFLAG;
   ResetConnections (true, TheFlag);
   InitConnectionLookup ();
